### ISSUE TITLE
Duplicate files in <head> when reloading a template with angular-ui-router

### ISSUE THREAD

**Issue by inigo001** [ocombe/ocLazyLoad#191]
Hi and sorry if this question has been already asked.  I have checked the #38 issue but I still don't see it very clear.

I have two HTML _templates_ that I load using `<a href="">` links.  I use angular-ui-router to route, so the only thing that changes when clicking the link is the content of the ui-view.  Each template has a .js and .css file associated with it that I load using ocLazyLoad.

Everything goes smoothly until I decide to go back and forth between the two pages.  Everytime I click a link both .js and .css files get loaded again adding two duplicate lines of code to the `<head>`.  The app still runs without problems since it loads the component just the first time, but the files keep getting loaded making the `<head>` a mess.

Here is the code I use to load the files with the routing.

```javascript
angular
    .module('miApp')
    .config(config);

function config( $stateProvider , $locationProvider , $urlRouterProvider , $ocLazyLoadProvider 	) 
{

	$urlRouterProvider.otherwise("/");

    $stateProvider	
        .state('testState', {
        	url: "/mirar",
        	templateUrl: 'views/viewTest1.html',
        	controller: 'view1Ctrl as miCtrl',
        	resolve: { 
  				loadMyCtrl: ['$ocLazyLoad', function($ocLazyLoad) {
             		return $ocLazyLoad.load('scripts/controllers/view1Ctrl.js');
    			}],
    			loadMyCSS: ['$ocLazyLoad', function($ocLazyLoad) {
             		return $ocLazyLoad.load('styles/testStyle.css');
    			}],
 		 	}
        })
        .state('testState2', {
        	url: "/mirar2",
        	templateUrl: 'views/viewTest2.html',
        	controller: 'view2Ctrl as miCtrl',
        	resolve: { 
  				loadMyCtrl: ['$ocLazyLoad', function($ocLazyLoad) {
             		return $ocLazyLoad.load('scripts/controllers/view2Ctrl.js');
    			}],
    			loadMyCSS: ['$ocLazyLoad', function($ocLazyLoad) {
             		return $ocLazyLoad.load('styles/testStyle.css');
    			}],
 		 	}
 		});
    
    $locationProvider.html5Mode(true);

    $ocLazyLoadProvider.config({
      	debug: true
    });
}
```

Is there a solution or am I doing something wrong?

**Comment by ocombe** [comment]
Are you using ocLazyLoad 1.0.0 ? If so, could you check if you have the same problem with beta2? https://github.com/ocombe/ocLazyLoad/releases/tag/1.0.0-beta.2
I have my idea on where the problem might be.

**Comment by ocombe** [comment]
Ok I checked it on my own, and I know where the problem is, I'll fix this asap :)

**Comment by inigo001** [comment]
Yep, tested it on beta2 and worked fine.

Thanks a lot for your great work!

**Comment by ocombe** [comment]
Fixed in 1.0.1 !

**Comment by rmayrink** [comment]
I'm using v1.0.4 and I'm with same problem, you can help me?
````javascript
	angular.module('App').config( function ( $stateProvider, $urlRouterProvider , $locationProvider) {

		$urlRouterProvider.otherwise("/login");

		$stateProvider
			.state({
				name: 'login',
				url: '/login',
				templateUrl: 'app/components/login/loginView.html',
                resolve: {
                    module: function($ocLazyLoad) {
                        return $ocLazyLoad.load([
                            'App.login'
                        ]);
                    }
                },				               
				controller: 'loginController'
			})
            .state({
                name: 'logout',
                url: '/logout',
                resolve: {
                    module: function($ocLazyLoad) {
                        return $ocLazyLoad.load([
                            'App.logout'
                        ]);
                    }
                },                             
                controller: 'logoutController'
            })            
			.state({
				name: 'dashboard',
				url: '/dashboard',
				templateUrl: 'app/components/dashboard/dashboardView.html',
                resolve: {
                    module: function($ocLazyLoad) {
                        return $ocLazyLoad.load([
                            'App.core.navside',
                            'App.core.toolbar',
                            'App.dashboard'
                        ]);
                    }
                },
				controller: 'dashboardController'
			})
			.state({
				name: 'employee',
				url: '/employee',
				templateUrl: 'app/components/dashboard-employee/read/readView.html',
                resolve: {
                    module: function($ocLazyLoad) {
                        return $ocLazyLoad.load([
                            'App.core.navside',
                            'App.core.toolbar',
                            'App.core.buttonFloat',
                            'App.dashboard.employee.Read'
                        ]);
                    }
                },				               
				controller: 'ReadController'
			})
...
````

My modules are:

````javascript
angular.module('App').config(function($ocLazyLoadProvider) {
		$ocLazyLoadProvider.config({
			debug: true,
			modules: [
				{
					name: 'App.core.navside',
					serie: true,
					files: [
						'app/shared/navside/navsideDirective.js',
						'app/shared/navside/navsideController.js'
					]
				},
				{
					name: 'App.core.toolbar',
					serie: true,
					files: [
						'app/shared/toolbar/toolbarDirective.js'
					]
				},	
				{
					name: 'App.core.buttonFloat',
					serie: true,
					files: [
						//'node_modules/ng-material-floating-button/mfb/src/mfb.js',
						'node_modules/ng-material-floating-button/src/mfb-directive.js',
						'node_modules/ng-material-floating-button/mfb/dist/mfb.min.css',						
					]
				},				
				{
					name: 'App.login',
					serie: true,
					files: [
					    'app/shared/data/user/userService.js',                       
						'app/components/login/loginController.js'
					]
				},
				{

					name: 'App.dashboard',
					serie: true,
					files: [
						'app/shared/data/user/userService.js',
						'app/shared/data/unit/unitService.js',
						'app/components/dashboard/dashboardController.js'
					]
				},
				{
					name: 'App.dashboard.employee.Read',
					serie: true,
					files: [
						'app/shared/data/user/userService.js',
						'app/shared/data/unit/unitService.js',
						'app/components/dashboard-employee/read/readController.js'
					]
				},
				{
					name: 'App.dashboard.employee.Create',
					serie: true,				
					files: [
						'node_modules/moment/min/moment.min.js',
						'node_modules/moment/locale/pt-br.js',
						'app/shared/data/user/userService.js',
						'app/shared/data/unit/unitService.js',
						'app/components/dashboard-employee/create/createController.js'
					]
				},

...
````
